.github/SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+## Supported Versions
+
+| Version       | Supported          |
+| ------------- | ------------------ |
+| 1.1.1         | :white_check_mark: |
+| < 1.1.1       | :x:                |
```

Comment:
Does this mean that #1-3 below are only supported in 1.1.1? Should we move this table following the next session `How we do security`?

Reply:
This means only 1.1.1 will be applicable for security patches